package.json

```diff
@@ -787,7 +787,7 @@
         },
         "vim.vimrc.path": {
           "type": "string",
-          "description": "Path to a Vim configuration file. If unset, it will check for $HOME/.vimrc or $HOME/_vimrc."
+          "description": "Path to a Vim configuration file. If unset, it will check for $HOME/.vimrc, $HOME/_vimrc, or $HOME/.config/nvim/init.vim, in that order."
```

Comment:
Sorry, this should also be `and`, not `or`

Reply:
Kinda confused by that.  If `$HOME/.vimrc` exists it WON'T check for `init.vim`.  So how is it "and"?

Reply:
On second thought, I don't really care lol